assets/js/blocks/cart/inner-blocks/cart-order-summary-heading/attributes.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
+	content: {
+		type: 'string',
+		default: __( 'Heading', 'woo-gutenberg-products-block' ),
+	},
+	lock: {
+		type: 'object',
+		default: {
+			remove: false,
+			move: false,
```

Comment:
After internal discussions it was decided that  we can leave it since it’ll be up to the user to remove it. And he can add a custom CSS to add some margin to fix the alignment issue.